TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -980,6 +1044,117 @@ see <doc:Functions#In-Out-Parameters>.
   ```
 -->
 
+#### Borrowing and Consuming Parameters
+
+The Swift compiler has varying ownership rules that it
+can use to manage object lifetimes across function calls.
+The default rules work well for most cases,
+but if necessary, you can use one of the `borrowing`
+or `consuming` modifiers to choose a specific ownership
+rule for a particular argument.
```

Comment:
Can we give more specific guidance than "when necessary"?  From your note below, it sounds like this is needed only for performance reasons.

Reply:
Clarified a bit.  Let me know if you think this still needs more.